packages/kbn-i18n-react/src/inject.tsx

```diff
@@ -12,5 +12,4 @@
  * More docs and examples can be found here https://github.com/yahoo/react-intl/wiki/API#injection-api
  */
 
-// eslint-disable-next-line @kbn/eslint/module_migration
-export { injectIntl as injectI18n } from 'react-intl';
+export { injectIntl as injectI18n, useIntl as useI18n } from 'react-intl';
```

Comment:
Is `useI18n` going to be used immediately? If not, I would suggest we delay exposing it until we really need it. WDYT?

Reply:
If we never expose it then it'll never be used. I would like to have it exposed and for people to start using it even instead of `injectI18n` (hooks > HOCs now adays :D )